docs/infrastructure/environments/target-roles/index.md

```diff
@@ -24,7 +24,7 @@ You can define as many environments, machines, and roles as you need; it all dep
 
 ## Creating a Role and Assigning it to a Tentacle {#MachineRoles-CreatingaroleandassigningittoaTentacle}
 
-Roles are created and saved in the database the moment you assign them to a Tentacle. To do so, just:
+Roles are created and saved in the database the moment you assign them to a Tentacle. Decide on the naming convention you will use, before creating your first role as it is not possible to change the case after the role has been created, for instance, all lowercase to camel case.
```

Comment:
It might be nice to use `deployment target` instead of `Tentacle` here. This might also benefit with a link to the UserVoice suggestion: http://octopusdeploy.uservoice.com/forums/170787/suggestions/3271170

Reply:
Thanks guys. Great suggestion about updating tentacles to deployment targets. I've added that change as well.